stargazer/stargazer.py

```diff
@@ -216,6 +217,10 @@ def append_notes(self, append):
         assert type(append) == bool, 'Please input True/False'
         self.notes_append = append
 
+    def table_label(self, table_label):
+        assert type(table_label) == str, 'Please input a string to use as the table label'
+        self.table_label = table_label
```

Comment:
I don't think naming the argument like the function can work ;-)

In any case, I suggest not creating any function in this case, setting the attribute should suffice.

Reply:
Ah you're right. We need tests!

If the general idea (before moving to the new API) is that more features can be configured by setting attributes directly, rather than exposed functions, should the documentation reflect this? Currently I think users would have to examine the code to know what else can be modified.

Reply:
> If the general idea (before moving to the new API) is that more features can be configured by setting attributes directly, rather than exposed functions, should the documentation reflect this? Currently I think users would have to examine the code to know what else can be modified.

You are right we should improve the docs... but this is probably worth doing after merging #44 .